Plugins/InitAspNetDatabase/Program.cs

```diff
@@ -37,7 +36,10 @@ static int Main(string[] args)
             string errorMessage = null;
             try
             {
-                Paths.InitializeRhetosServerRootPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\.."));
+                var rootPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..");
+                var configurationProvider = new Rhetos.ConfigurationBuilder().
+                    AddRhetosAppConfiguration(rootPath).Build();

```

Comment:
Code style: redundant `Rhetos` namespace qualifier, line breaks. Should read something like this:
```
            var configurationProvider = new ConfigurationBuilder()
                .AddRhetosAppConfiguration(rootPath)
                .Build();
```